api/controllers/adoptionFormController.js

```diff
@@ -0,0 +1,68 @@
+const mongoose = require('mongoose');
+const {adoptionForm, validateAdoptionForm} = require('../models/adoptionForm');
+
+// moge tak użyć?
+app.use('/adoptionForms/:adoptionId', (req, res, next) => {
+    const adoptionFormId = Number(req.params.adoptionId);
+    const adoptionFormIndex = adoptionForms.findIndex(form => form.id === adoptionFormId);
+    if (cardIndex === -1) {
+      return res.status(404).send('Card not found');
+    }
+    req.adoptionFormIndex = adoptionFormIndex;
+    next();
+  });
+
+// GETL ALL adoption forms
+exports.AdoptionFormGetAll = async (req, res, next) => {
+    const adoptionForms = await adoptionForm.find();
+    res.send(adoptionForms);
+}
+
+// GET one adoption form
+exports.AdoptionFormGetOne = async (req, res, next) => {
+    const adoptionFormsOne = await adoptionForm.findById(req.params.adoptionId);
+    if(!adoptionFormsOne) return res.status(404).send('Formularz wolontariusza o podanym ID nie istnieje.');
+
+    res.send(adoptionFormsOne);
+}
+
+// POST adoption form
+exports.addAdoptionForm = async (req, res, next) => {
+    try{
+        const {content, userID, animalID} = req.body;
+        const value = await validateAdoptionForm.validateAsync({
```

Comment:
Popraw sobie funkcję do walidacji. bo w Postman wyświetla się błąd "validateAdoptionForm.validateAsync is not a function"